src/components/Card.js

```diff
@@ -22,23 +26,32 @@ export default class Card {
 
   _setEventListeners() {
     this._likeButton.addEventListener("click", () => {
-        this._handleLikeIcon();
-      });
+      this._handleLikeIcon(this._id, this._isLiked);
+    });
 
     this._deleteButton.addEventListener("click", () => {
-        this._handleDeleteCard();
+        this._handleDeleteCard(this._id);
     });
 
     this._cardImageElement.addEventListener('click', () => {
       this._handleImageClick(this._name, this._link);
     });
   }
 
-  _handleLikeIcon() {
-    this._likeButton.classList.toggle(cardselectors.cardLikeToggle);
+  _renderLikeIcon() {
```

Comment:
The logic for updating and rendering likes looks brilliant!

Reply:
Thank you!